_activities/sinkfloat.md

```diff
@@ -14,12 +14,12 @@ image:
    thumb:
    title:
 downloads:
-   -
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)